lib/cloud-metadata/index.js

```diff
@@ -71,18 +55,24 @@ class CloudMetadata {
     // normalize arguments
     if (!cb) {
       cb = config
-      config = DEFAULT_PROVIDER_CONFIG
```

Comment:
The block comment for this function should be updated to drop the mentions of DEFAULT_PROVIDER_CONFIG.

Reply:
this is why you don't comment your functions ;)  Comment updated to remove old variable reference.